pkgs/development/python-modules/asyncstdlib-fw/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # build-system
+  pdm-backend,
+}:
+
+buildPythonPackage rec {
+  pname = "asyncstdlib-fw";
+  version = "3.13.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "asyncstdlib_fw";
+    inherit version;
+    hash = "sha256-Ua0JTCBMWTbDBA84wy/W1UmzkcmA8h8foJW2X7aAah8=";
+  };
```

Comment:
Can you add a comment stating why we can't use the GH source here?

Reply:
Added.

Reply:
Try now. I dropped the whole PR into draft while I was checking everything and running nixpkgs-review. Trying to minimize the number of iterations for you.

Reply:
This is perfectly fine thanks! But in this case, try not to answer "fixed" before having actually pushed the changes ;)